src/gerbil/runtime/thread.ss

```diff
@@ -10,14 +10,20 @@ namespace: #f
 
 ;; spawn an actor thread apply f to args
 (def (spawn f . args)
-  (spawn-actor f args #!void #f))
+  (unless (procedure? f)
```

Comment:
Use `check-argument` ? (also below)

Reply:
we dont have it down here.

Reply:
I actually have check-procedure in ./error,  let ne use that.